PAS_Database/dbo/Stored Procedures/Procs1/SearchCustomerInvoicesByCustId.sql

```diff
@@ -57,7 +57,7 @@ BEGIN
     
 		SELECT SOBI.SalesOrderId AS 'Id',      
 	         SOBI.SOBillingInvoicingId AS 'SOBillingInvoicingId',       
-		      CASE WHEN SOBI.IsProforma = 1 THEN 'Proforma Invoice' ELSE 'Invoice' END  AS 'DocumentType',      
```

Comment:
Why we change this ?

Reply:
its client requirement to say it Proforma Invoice to only Invoice